harness/determined/keras/_load.py

```diff
@@ -3,17 +3,24 @@
 import pathlib
 from typing import List, Optional, cast
 
-from tensorflow.python.training.tracking import tracking
+import tensorflow as tf
+from packaging import version
 
 import determined as det
 from determined import keras
 
+# In TF 2.14, tracking module was removed.
+if version.parse(tf.__version__) >= version.parse("2.14.0"):
+    from tensorflow.python.trackable.autotrackable import AutoTrackable  # noqa: I2041
```

Comment:
we have a code style guideline where we don't want to use object names directly without modules.

Can you change both of these imports to something like:
`from tensorflow.python.trackable import autotrackable as tf_tracking`

Then you can reference the object as `tf_tracking.AutoTrackable`